python/lsst/pipe/tasks/multiBandUtils.py

```diff
@@ -144,7 +144,7 @@ def getShortFilterName(name):
     # I'm not sure if this is the way this is supposed to be implemented, but it seems to work,
     # and its the only way I could get it to work.
     try:
-        return afwImage.Filter(name).getFilterProperty().getName()
+        return afwImage.Filter(name).getName()
```

Comment:
I believe this should be `getCanonicalName()`; see the [definition of getFilterProperty()](https://github.com/lsst/afw/blob/master/src/image/Filter.cc#L359-L360).

Reply:
In fact, looking at the function definition, I'm pretty sure this could be entirely removed. Maybe remove the existing comment and mark it as TODO for DM-27170?